.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      # strikt 0.34.1 introduced breaking changes to .isNotNull()
+      - dependency-name: "io.strikt:strikt-core"
```

Comment:
and what prevents us from updating the code that broke?

Reply:
I don't have a solution, I think it's an open issue they haven't fixed yet. I'm not quite sure.

Reply:
So I bumped the version back up to 1.34.1 and changed assertions to assert nothing was thrown instead. They changed some types on extension functions that caused `isNotNull()` to only work for nullable types, and my test was constructing something non-nullable. The assertion was not necessary anyway since I just wanted to make sure the builder succeeded and didn't throw - admittedly not a very valuable test.